Gemfile.lock

```diff
@@ -38,6 +38,7 @@ GEM
     httpclient (2.8.3)
     httpx (0.23.3)
       http-2-next (>= 0.4.1)
+    jdbc-mysql (8.0.30)
```

Comment:
This all looks like it changed because of something to do with jruby, but if it works, it works.